support/dynamics/gp/unable-to-generate-next-payment-or-voucher-number-when-entering-payables-transaction.md

```diff
@@ -96,25 +72,40 @@ To resolve this problem, enter a larger number in the **Next Payment Number/Vouc
       select max (CNTRLNUM) from PM00400 where CNTRLTYP = 1
       ```
 
-### Resolution 3
+## Cause 3
+
+The **Next Payment Number/Voucher Number** field does not start with leading zeros.
+
+#### Resolution
 
-To resolve this problem, make sure that the number in the **Next Payment Number/Voucher Number** field in the Payables Setup Options window starts with leading zeros. For example, if the number starts with 999, change the number to 0000000999.
+To resolve this problem, make sure that the number in the **Next Payment Number/Voucher Number** field in the **Payables Setup Options** window starts with leading zeros. For example, if the number starts with 999, change the number to 0000000999.
 
-### Resolution 4
+> [!TIP]
+> To open the **Payables Setup Options** window, on the Microsoft Dynamics GP menu, point to **Tools** > **Setup** > **Purchasing**, select **Payables**, and then select the **Options** button.
 
-To resolve this problem, make sure that the number in the **Next Payment Number/Voucher Number** field in the Payables Setup Options window has no letters or alpha-numeric characters.
+## Cause 4
 
-### Resolution 5
+The **Next Payment Number/Voucher Number** field contains letters.
+
+#### Resolution
+
+To resolve this problem, make sure that the number in the **Next Payment Number/Voucher Number** field in the **Payables Setup Options** window has no letters or alpha-numeric characters.
+
+> [!TIP]
+> To open the **Payables Setup Options** window, on the Microsoft Dynamics GP menu, point to **Tools** > **Setup** > **Purchasing**, select **Payables**, and then select the **Options** button.
+
+## Cause 5
+
+Data in the Payables Transaction Logical File group is missing or is corrupted.
+
+### Resolution 
 
 To resolve this problem, perform the Check Links routine on the Payables Transaction Logical File. Have all users sign out of Microsoft Dynamics GP before proceeding. To do this, follow these steps.
 
 > [!WARNING]
 > If you use Microsoft Dynamics GP 9.0, you must install Service Pack 2 before you follow these steps. The Check Links process may incorrectly remove applied records if you run Check Links in Microsoft Dynamics GP 9.0 or in Microsoft Dynamics GP 9.0 Service Pack 1.
```

Comment:
Hi @theley502, can you help double check whether we need to keep this warning? Thanks!

Reply:
remove it, good catch